packages/core/upload/server/controllers/validation/admin/folder-file.js

```diff
@@ -75,7 +75,11 @@ const validateMoveFoldersNotInsideThemselvesSchema = yup
       });
 
       const unmovableFoldersNames = folders
-        .filter((folder) => destinationFolder.path.startsWith(folder.path))
+        .filter(
+          (folder) =>
```

Comment:
Is it worth lifting this into a util func seeing as it's used in the same way twice?

Reply:
makes sense! I will create a util function for this :) 

I am trying to fix the test, running it in local everything works ok, dont know why it doesnt work in github actions 🤔